docs/production-deployment/cloud/get-started/service-accounts.mdx

```diff
@@ -166,53 +166,53 @@ Example:
+As with regular Service Accounts, Namespace-scoped Service Accounts can be created using Temporal Cloud UI or tcld.
 
 #### Using the Cloud UI {#scoped-ui}
 
-Currently, creating a Namespace Scoped Service Account from the Temporal Cloud UI happens on an individual [Namespace](/cloud/namespaces#manage-namespaces) page.
+Currently, creating a Namespace-scoped Service Account from the Temporal Cloud UI happens on an individual [Namespace](/cloud/namespaces#manage-namespaces) page.
 If the current Namespace has API key authentication enabled, then there will be a `Generate API Key` button as a banner on the top of the Namespace page or in the `Authentication` section.
 
-By clicking on the `Generate API Key` button, a Namespace Scoped Service Account will be automatically created for the given Namespace (if one does not already exist) and an associated API key will be displayed. This key will have the maximum expiration time, which is 2 years.
+By clicking on the `Generate API Key` button, a Namespace-scoped Service Account will be automatically created for the given Namespace (if one does not already exist) and an associated API key will be displayed. This key will have the maximum expiration time, which is 2 years.
 
-The resulting Namespace Scoped Service Account will be named `<namespace>-service-account` and will have an `Admin` Namespace Permission by default.
+The resulting Namespace-scoped Service Account will be named `<namespace>-service-account` and will have an `Admin` Namespace permission by default.
 
 #### Using tcld
 
-To create a Namespace Scoped Service Account with tcld, use the `tcld service-accounted create-scoped` command:
+To create a Namespace-scoped Service Account with tcld, use the `tcld service-accounted create-scoped` command:
 
 ```
 tcld service-account created-scoped -n "test-scoped-sa" --np "test-ns=Admin"
 ```
 
-This example creates a Namespace Scoped Service Account for the Namespace `test-ns`, named `test-scoped-sa`, with `Admin` Namespace Permission.
-Note that the Account Role is omitted, since Namespace Scoped Service Accounts always have a `Read` Account Role.
+This example creates a Namespace-scoped Service Account for the Namespace `test-ns`, named `test-scoped-sa`, with `Admin` Namespace Permission.
+Note that the Account Role is omitted, since Namespace-scoped Service Accounts always have a `Read` Account Role.
 
 ### Lifecycle
 
-When a Namespace is deleted, all associated Namespace Scoped Service Accounts are automatically deleted as well.
-Therefore, you do not need to manually remove Namespace Scoped Service Accounts after deleting a Namespace.
+When a Namespace is deleted, all associated Namespace-scoped Service Accounts and their associated API keys are automatically deleted as well.
+Therefore, you do not need to manually remove Namespace-scoped Service Accounts and their API keys after deleting a Namespace.
```

Comment:
cc @shakeelrao to confirm

Reply:
yeppers.

Reply:
How about [this](https://github.com/temporalio/documentation/pull/3904/commits/fe1d97aa7bff7eff584c19c87e2f7e425582432b)? You can view it on the preview url [here](https://temporal-documentation-jkesc0xdy.preview.thundergun.io/cloud/service-accounts#lifecycle).

Reply:
I think this addition is better situated within the Delete section - i don't think it needs its own heading :)